pkg/planner/cascades/rule/binder.go

```diff
@@ -159,15 +153,15 @@ func NewBinder(p *pattern.Pattern, gE *memo.GroupExpression) *Binder {
 		// empty stack info, means the toppest loop.
 		// pre-set nil is for later alignment with the traceID indexing.
 		stackInfo: []*list.Element{},
-		holder:    &GroupExprHolder{Cur: gE},
+		holder:    gE,
```

Comment:
Should we directly keep the Logicalplan as the holder in here ?

Reply:
not possible, some unchanged holder should be able to be traced back to its original group, LogicalPlan couldn't do that